angular/projects/spark-angular/src/lib/components/sprk-modal/sprk-modal.component.ts

```diff
@@ -51,7 +51,7 @@ import * as _ from 'lodash';
         >
           <div
             *ngIf="modalType == 'wait'"
-            class="sprk-o-Stack__item sprk-c-Spinner sprk-c-Spinner--circle sprk-c-Spinner--large sprk-c-Spinner--dark"
+            class="sprk-o-Stack__item sprk-c-Spinner sprk-c-Spinner--circle sprk-c-Spinner--large sprk-c-Spinner--primary"
```

Comment:
hey @guern1kn , what are all the colors for spinners now? "primary" seems like it should be the default spinner color instead of a modifier if there is only two spinner colors.

Reply:
So, we have white, which is the default, dark is black and the primary modifier is red. I had named it sprk-c-Spinner--red, but we wanted to change that. Went with primary because red is our primary color. Can revisit this too.

Reply:
The thing we want to be careful here with is the use of --dark --light all over the place. I noticed you have it set to mean different things. For some things, its meaning is "this is a link that is dark in color but is on a light background" so its --light. Which is different than this context. where --dark actually means the element will be dark. Thats why for stepper you see "sprk-c-Stepper--has-dark-bg". Instead of sprk-c-Stepper--dark. Can you follow this pattern to make the naming consistent across the system?

Reply:
Lets chat about this off github. It'll be easier.